.pipelines/build.yml

```diff
@@ -29,6 +29,24 @@ steps:
     versionSpec: $(nugetVersion)
     checkLatest: true
 
+- task: DotNetCoreCLI@2  
+  inputs:
+    command: custom
+    custom: tool
+    arguments: install --tool-path . nbgv
+  displayName: Install NBGV tool
+  env:
+    COREHOST_TRACE: 0
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1    
+
+- script: nbgv cloud

```

Comment:
That's what I'd like to discuss further: combining build and release. That's what I'd rather do not do. Until there are good reasons. I'd like to learn more.

Reply:
This is effectively just communicating any version information set in the project to azure DevOps. So if building locally this isn't needed but the version information is still injected. This is the missing step you had issues with where it was essentially just the increment of build numbers instead of version number.

Reply:
What I'd like to have is YAML-based release definition but I don't see it's easily available/doable. They introduced something called multi-stage YAML-based build definitions instead, which suppose to replace release. Are you familiar with this area?